pkg/api/register.go

```diff
@@ -111,6 +111,15 @@ func AddToScheme(scheme *runtime.Scheme) {
 		&unversioned.APIGroup{},
 		&unversioned.APIResourceList{},
 	)
+
+	// Types registered here will be kept unversioned when they are encoded.
+	// It's for backward compatibility with release 1.1. clients.
+	Scheme.AddToKeepUnverioned(
```

Comment:
We have extensions endpoints in release-1.1, so probably we need to set APIVersion to "" for all groups.

I'll fix the typo.